apps/prairielearn/elements/pl-order-blocks/pl-order-blocks.py

```diff
@@ -410,6 +423,9 @@ def render(element_html, data):
         return html
 
     elif data["panel"] == "submission":
+        
+        feedback_type = pl.get_string_attrib(element, 'feedback', FEEDBACK_DEFAULT)
```

Comment:
Since you're using `feedback_type` as an enum with a fixed number of values, it would be best to do this explicitly along with the `get_enum_attrib` function.

Reply:
this was an already existing attribute, not new to this PR, so I'm not going to refactor to use `get_enum_attrib` as part of this PR. But it should be done in future when the typechecking is put back in to this element, along with converting a few of the other string attributes to Enums.